official/utils/misc/distribution_utils.py

```diff
@@ -56,6 +58,10 @@ def get_distribution_strategy(distribution_strategy="default",
                        "cannot be set to 'off'.".format(num_gpus))
     return None
 
+  if distribution_strategy == "collective" or num_workers > 1:
```

Comment:
is 1 remote worker a valid use case? which strategy do we want users to use in that case? 

also, will it be confusing if the user explicitly says "collective" with 1 worker, but we still fallback to mirrored strategy? At the very least we should say if this is the case in documentation.

Reply:
The current behavior is that we use `CollectiveAllReduceStrategy` either when user explicitly specifies `distribution_strategy=collective` or there is more than 1 worker.  So, if they specify `collective` with 1 worker it should use `CollectiveAllReduceStrategy` and not `MirroredStrategy`.

Reply:
oh my bad. I think I must've read that "or" as an "and". What you have is fine then.